sdk/cosmos/spark.yml

```diff
@@ -22,4 +22,4 @@ stages:
       DatabricksEndpoint: $(spark-databricks-endpoint)
       DatabricksToken: $(spark-databricks-token)
       SparkVersion: 'azure-cosmos-spark_3-2_2-12'
-      ClusterName: 'oltp-ci-spark32-2workers-ds3v2'
\ No newline at end of file
+      ClusterName: 'oltp-ci-spark32-2workers-ds3v2'
```

Comment:
working item to add databricks integration tests for 3.3: https://github.com/Azure/azure-sdk-for-java/issues/31834